src/index.html

```diff
@@ -11,6 +11,35 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+      action="#"
+      method="POST"
+      class="form"
+      data-qa="big"
+    >
+      <input
+        name="searchInput"
+        class="form__input"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+      >
+      <button type="submit" class="form__btn"></button>
+    </form>
+
+    <form
+      action="#"
+      method="POST"
+      class="form"
```

Comment:
I think better sollution would be to style small element using modifier class.
You can add
```suggestion
      class="form form--small"
```
Instead of relying on **data-qa** for css selector which can change.